src/loaddb/load_server_loader.cpp

```diff
@@ -454,12 +454,14 @@ namespace cubload
 
     int error_code = locator_multi_insert_force (m_thread_ref, &m_scancache.node.hfid, &m_scancache.node.class_oid,
 		     m_recdes_collected, true, op_type, &m_scancache, &force_count, pruning_type, NULL, NULL,
-		     UPDATE_INPLACE_NONE, true);
+		     UPDATE_INPLACE_NONE, true, &records_inserted);
```

Comment:
do we need to add a new argument to locator_multi_insert_force? it is same as m_recdes_collected.size ().